specification/metadata/providerprefix.md

```diff
@@ -0,0 +1,17 @@
+# Provider Prefix
+
+Used as the prefix for the name of custom columns, useful for the consumer of the data to quickly identify custom column extensions to the FOCUS schema implemented by the provider.
```

Comment:
REMINDER: 
align with/adjust the following column naming conventions requirement:

> All custom columns MUST be prefixed with a consistent x_ prefix to identify them as external, custom columns and distinguish them from FOCUS columns to avoid conflicts in future releases.

Reply:
I feel it's worth discussing if x_ is better than $ProviderPrefix$_ (now that we have a way to get that).

Reply:
oh I need to re-read that section I had thought we went with a $ProviderPrefix$_. Let's decide if we are going to reopen this conversation or just align this to the current x_ at this weeks maintainers meeting.